docs/insomnia/subscription-management.md

```diff
@@ -5,6 +5,18 @@ category: "Insomnia Subscriptions"
 category-url: subscription-management
 ---
 
+## View Subscription Details
+
+Users can view their current plan, the total number of seats in the subscription, and the number of seats currently in use by navigating to the—depending on your plan—"Account Settings" or "Enterprise Controls" page and then to the "Billing" section.
```

Comment:
I was only able to test this myself for the Enterprise plan, but from what I could put together from the code and existing documentation it seems like the same information should be available under the Account Settings for non-Enterprise users.

Reply:
Yes